site/collections/CollectionsPage.scss

```diff
@@ -0,0 +1,26 @@
+.collections-page {
+    background-color: $gray-10;
+
+    .collections-page__header {
+        background-color: #fff;
+        margin-bottom: 24px;
+    }
+    .collection-title {
+        margin-bottom: 0;
+    }
+    .collection-explanation {
+        color: $blue-60;
+        margin-bottom: 32px;
+
+        &:not(:last-of-type) {
+            margin-bottom: 0;
+        }
+    }
+
+    figure {
+        width: 100%;
+        height: 650px;
```

Comment:
Can we use `$grapher-height` here?

Would also be nice to use the height of the viewport on mobile.